uk_bin_collection/uk_bin_collection/councils/SouthendOnSeaCityCouncil.py

```
@@ -0,0 +1,58 @@
+    implementation.
+    """
+
+    def parse_data(self, page: str, **kwargs) -> dict:
+        data = {"bins": []}
+
+        user_uprn = kwargs.get("uprn")
+        check_uprn(user_uprn)
+
+        api_url = f"https://apps.cloud9technologies.com/southend/citizenmobile/openapi/wastecollections/{user_uprn}"
+
+        response = requests.get(api_url)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)